index.html

```diff
@@ -8,16 +8,24 @@
         <meta name="author"
               content="Fredrik Lundh, Jeffrey A. Clark (Alex), Pillow Contributors">
 
+        <link rel="stylesheet" href="assets/css/pillow.css">
+
+        <link href="assets/fontawesome/css/fontawesome.css" rel="stylesheet">
+        <link href="assets/fontawesome/css/brands.css" rel="stylesheet">
+        <link href="assets/fontawesome/css/solid.css" rel="stylesheet">
```

Comment:
Shall we use the minimised ones? 
```suggestion
        <link href="assets/fontawesome/css/fontawesome.min.css" rel="stylesheet">
        <link href="assets/fontawesome/css/brands.min.css" rel="stylesheet">
        <link href="assets/fontawesome/css/solid.min.css" rel="stylesheet">
```